app/services/katello/pulp3/repository_mirror.rb

```diff
@@ -145,6 +146,7 @@ def refresh_distributions(options = {})
         dist_params = {}
         dist_params[:publication] = options[:publication] if options[:publication]
         dist_params[:repository_version] = version_href if options[:use_repository_version]
+        dist_params[:content_guard_href] = options[:content_guard_href] if options[:content_guard_href]
```

Comment:
would it make more sense to 'lookup' the content guard href here and set it if needed?

Reply:
the other options on the distributor are created as part of the distribution_options method in this file